scala3doc/src/dotty/dokka/transformers/ImplicitMembersExtensionTransformer.scala

```diff
@@ -17,13 +17,13 @@ class ImplicitMembersExtensionTransformer(ctx: DokkaContext) extends Documentabl
 
     def expandMember(outerMembers: Seq[Member])(c: Member): Member =
       val companion = c match
-        case classlike: DClass => ClasslikeExtension.getFrom(classlike).flatMap(_.companion).map(classlikeMap)
+        case classlike: DClass => ClasslikeExtension.getFrom(classlike).flatMap(_.companion).flatMap(classlikeMap.get)
```

Comment:
Is it expected that `classlikeMap` would miss some classlikes?

Reply:
ahh, actually I changed it on the beginning of implementing this PR because it was throwing exceptions, but I've checked and this change is unnecessary